README.rst

```diff
@@ -18,7 +18,7 @@ for starkit in the following way. This will create an environment called `starki
     # you can change the -n argument
     
     conda env create --file starkit_env27.yml -n starkit
-    source activate starkit
+    conda activate starkit
```

Comment:
that does not work with all anaconda/miniconda installations. In fact the majority still uses `source activate ...`. can you give both options?

Reply:
OK NP.
As you are suggesting, I am giving both options. `conda activate` as a comment preceded by '#'.